resume.md

```diff
@@ -1,49 +1,65 @@
-# FIRST LAST NAME
+# Armen Phelps
 
-(123) 456 - 7890 | email@gmail.com
+(510) 846 - 6136 | armenphelps204@berkeley.edu
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+https://www.linkedin.com/in/armenphelps2022/
```

Comment:
would still include your github + you can use the short version of the url insteda of `https://www.`